client/hosting/server-settings/components/defensive-mode-card/index.tsx

```diff
@@ -0,0 +1,146 @@
+import { Button, FormLabel } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import moment from 'moment';
+import { useState } from 'react';
+import FormSelect from 'calypso/components/forms/form-select';
+import { HostingCard, HostingCardDescription } from 'calypso/components/hosting-card';
+import InlineSupportLink from 'calypso/components/inline-support-link';
+import {
+	useEdgeCacheDefensiveModeMutation,
+	useEdgeCacheDefensiveModeQuery,
+	useEdgeCacheQuery,
+} from 'calypso/data/hosting/use-cache';
+import { useSelector } from 'calypso/state';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+import './style.scss';
+
+type DefensiveModeCardProps = {
+	disabled?: boolean;
+};
+
+const AVAILABLE_TTLS = [
+	{
+		label: '1h',
+		value: 3600,
+	},
+	{
+		label: '12h',
+		value: 43200,
+	},
+	{
+		label: '24h',
+		value: 86400,
+	},
+	{
+		label: '2d',
+		value: 172800,
+	},
+	{
+		label: '7d',
+		value: 604800,
+	},
+];
+
+export default function DefensiveModeCard( { disabled }: DefensiveModeCardProps ) {
+	const translate = useTranslate();
+	const siteId = useSelector( getSelectedSiteId );
+	const [ ttl, setTtl ] = useState( AVAILABLE_TTLS[ 0 ].value );
+
+	const { data: isEdgeCacheActive } = useEdgeCacheQuery( siteId );
+	const { data: defensiveModeData } = useEdgeCacheDefensiveModeQuery( siteId );
+	const { mutate, isPending: isEnabling } = useEdgeCacheDefensiveModeMutation( siteId );
+
+	const enabledUntil = moment.unix( defensiveModeData?.enabled_until ?? 0 );
+
+	return (
+		<HostingCard className="defensive-mode-card" title={ translate( 'Defensive mode' ) }>
+			<HostingCardDescription>
+				{ translate(
+					'Extra protection against spam bots and attacks. Visitors will see a quick loading page while we run additional security checks. {{a}}Learn more{{/a}}',
+					{
+						components: {
+							a: <InlineSupportLink supportContext="hosting-defensive-mode" showIcon={ false } />,
```

Comment:
The `Learn more` link opened the Global Edge Cache support doc, but I would expect to see something that explains what defensive mode is and how to enable/disable it.